src/app/component/key/key-interface.js

```diff
@@ -0,0 +1,294 @@
+import './key-interface.css';
+import './key-interface.media.css';
+import createElement from '../../utils/create-element';
+import TyperInterface from './typer-interface';
+
+const CssClasses = {
+  BLOCK: 'key',
+  WIDE: 'key_semiwide',
+  SHELL: 'key__shell',
+  SHELL_ACTIVE: 'key__shell_active',
+  SHELL_STRETCH: 'key__shell_stretch',
+  SHELL_M_SP2: 'key__shell_media_span2',
+  SH: {
+    spec: 'key__shell_type_spec',
+    letter: 'key__shell_type_letter',
+    punct: 'key__shell_type_punct',
+  },
+  DESCRIPT: 'key__desc',
+  DESC: {
+    spec: 'key__desc_type_spec',
+    letterMain: 'key__desc_type_letter-main',
+    letterAlt: 'key__desc_type_letter-alt',
+    punctMain: 'key__desc_type_punct-main',
+    punctAlt: 'key__desc_type_punct-alt',
+  },
+};
+
+class KeyInterface {
```

Comment:
This class should be divided into many ones. This class should be used only for keys logic, but data should be stored in separate class.
Classes should not have `Interface` in its name. `KeyComponent` is OK.